docs/i18n-languages/portuguese/CONTRIBUTING.md

```diff
@@ -57,7 +57,7 @@ Nós agradecemos sua ajuda na escrita desses artigos. Você não precisa ser um
 
 Para ajudar, encontre um `artigo provisório` na [página de Guias](https://www.freecodecamp.org/guide), escreva o artigo, e então abra um <i>pull request</i> (PR) para substituir o provisório pelo seu artigo. Um [<i>pull request</i>](https://help.github.com/articles/about-pull-requests/) é a forma como você vai sugerir mudanças. Ele permite que outros saibam das suas mudanças, revisem-nas e as adotem.
 
-Se você não consegue encontrar um artigo provisório do tópico sobre o qual você gostaria de escrever, você pode abrir um PR que cria o artigo provisório e então incluir o rascunho do seu artigo.
```

Comment:
Need to updated the markdonw headings in above session "Contribua para essa base de códigos abertos"